README.md

```diff
@@ -3,10 +3,10 @@
 
 ## Quick Start:
 
-Install the dependencies necessary to run the tool.
+This runs on Node with some dependencies. Download, unzip and then go to the folder where `package.json` is and run `npm install` to install the dependencies necessary to run the tool.
```

Comment:
If you clone the repo you don't have to unzip the folder, also `npm install` can be run from anywhere within the directory, not just the folder with package.json. Can you rephrase this as "Download or clone the repo, navigate into the directory, and run `npm install`"?

Reply:
Thanks! I'm not very familiar with Node and, as usual with unknown environments, I often struggle to start because I miss the obvious preliminary steps ´:-)  

I hope this helps noobs like me